projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -41,6 +41,13 @@ export const mapStateToSummaryFeatureProps = ( state, featureSlug ) => {
 				summaryActivateButtonLabel: __( 'Enable', 'jetpack' ),
 				configLink: '#/settings?term=cdn',
 			};
+		case 'publicize':
+			return {
+				configureButtonLabel: __( 'Settings', 'jetpack' ),
+				displayName: __( 'Publicize', 'jetpack' ),
+				summaryActivateButtonLabel: __( 'Enable', 'jetpack' ),
+				configLink: '#/settings?term=publicize',
```

Comment:
Should we point to Calypso in this case, where folks will be able to connect new services to their site? `getRedirectUrl` should be helpful for this:
https://github.com/Automattic/jetpack/blob/39fa657bfe0aaca179275331752fd99a74ac1c41/projects/plugins/jetpack/_inc/client/sharing/publicize.jsx#L49

Reply:
Good idea, much more helpful to point to Calypso here.